arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts

```diff
@@ -159,6 +159,11 @@
 				BCM2835_PLLH_AUX
 				BCM2835_PLLH_PIX
 			>;
+
+	fragment@19 {
```

Comment:
I don't know the .dts format.
Is the number "19" in `fragment@19` important which it is? In the original commit it is `fragment@17` but here `fragment@17`is already in use, so I edit it to "19". Could it be a problem? @ikalkov

Reply:
AFAIK, fragments can have different indices, but please read https://octavosystems.com/app_notes/osd335x-design-tutorial/osd335x-lesson-2-minimal-linux-boot/linux-device-tree-overlay/ and understand how it works. You should also check if such node exists in the original DTS file you are overriding.